### PR TITLE
Tms 6

### PR DESCRIPTION
**Pull request by gnirula**

### CODE REVIEW

**Review by kishoreRajput570 — APPROVED** [review]

**Comment by kishoreRajput570** [comment]
@gnirula please resolve the conflicts

**Comment by gnirula** [comment]
I have resolved the conflicts but the index.js in the development branch might further create a problem at the time of adding headers and footers. @kishoreRajput570

File: src/index.js [hunk 1]
```diff
@@ -2,17 +2,18 @@ import React from 'react';
...
+import Signin from './Signin'
 import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <BrowserRouter>
-  <App />
+  {/* <App /> */}
+  <Signin />
```
Comment (kritikasinglaincedo) [inline src/index.js]: Please use App.js component instead of directly calling Signin Component

**Review by kritikasinglaincedo — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (gnirula) [reply src/index.js]: I have already fixed that issue the file you are viewing is outdated.

**Pull request merged**